Neos.Neos/Classes/Domain/Model/User.php

```diff
@@ -36,6 +36,11 @@ class User extends Person implements UserInterface
      */
     protected $preferences;
 
+    /**
+     * @var string
+     */
+    protected $Persistence_Object_Identifier;
```

Comment:
```suggestion
    /**
     * This property will be introduced via Flows persistence magic aspect
     * We declare it here already as workaround to not have to use {@see PersistenceManager::getIdentifierByObject()} to access this identifier.
     * @var string
     */
    protected $Persistence_Object_Identifier;
```

nice "hack" idea ... :D I approve ... probably better than injecting the persistence manager to do the same? But the above property should get a nice doc comment